include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -103,6 +140,13 @@ std::string sanitizePrefix(std::string prefix) {
     }
     return prefix;
 }
+
+// All supported metric types
+constexpr char METRIC_TYPE_COUNT[] = "c";
+constexpr char METRIC_TYPE_GUAGE[] = "g";
+constexpr char METRIC_TYPE_TIMING[] = "ms";
+constexpr char METRIC_TYPE_HISTOGRAM[] = "h";
```

Comment:
Do you have a link that details how histogram should be used? I didn't find them in the [metric types doc](https://github.com/statsd/statsd/blob/master/docs/metric_types.md) (that we should probably link in the README somewhere actually) and could only find a mention in [gostatsd](https://github.com/atlassian/gostatsd#timer-histograms-experimental-feature).

Reply:
i forget where i first read about it but looking at the code for the 100th time it seems like it would get stored as a counter, which doesnt seem right: https://github.com/statsd/statsd/blob/9cf77d87855bcb69a8663135f59aa23825db9797/stats.js#L317-L322

so ill remove it

Reply:
done